website/docs/cookbooks/test-only.md

```diff
@@ -0,0 +1,145 @@
+---
+title: Filter the test suites to run
+sidebar_position: 11
+---
+
+The `test-only` parameter it is a useful option for when you only want to run a specific subset of tests.
+
+In this cookbook, we will walk through how to use the `test-only` parameter with the `test` sub-command in Scala CLI,
+specifically when using the `munit` and `utest` test frameworks.
+
+## Filter the test suites
+
+The `test-only` parameter in Scala CLI supports using glob patterns to filter test suites to run. A glob pattern is a
+string which contains asterisk `*` characters to match a set of test suites.
+
+There are three examples of glob patterns how to filter test suites with the `test-only`:
+
+- start with `test` - `test*`
+- end with `test` - `*test`
+- contains `test` -  `*test*`
```

Comment:
would the forth not be `.` I think . replaces a single char instead of * replacing 0+ ?

Reply:
```
➜  scala-demo cat Test.scala 
//> using lib "org.scalameta::munit::0.7.29"

class Tests extends munit.FunSuite {
  test("foo") {
    assert(2 + 3 == 5)
  }
}
➜  scala-demo scala-cli test .  -- '*foo' 

Tests:
  + foo 0.039s
➜  scala-demo scala-cli test .  -- '*fo.' 

Tests:
```

In my tests `.` isn't matched to any character.

Reply:
Yes, it seems that `.` isn't treated as a special character.